lib/config/presets/internal/group.ts

```diff
@@ -698,6 +698,15 @@ const staticGroups = {
       },
     ],
   },
+  vaadinWebComponents: {
+    description: 'Group all Vaadin Web components together.',
+    packageRules: [
+      {
+        groupName: 'Vaadin Web components',
+        matchSourceUrls: ['https://github.com/vaadin/web-components'],
+      },
+    ],
+  },
```

Comment:
move to this list instead:
https://github.com/renovatebot/renovate/blob/645f5d519493d8cd8d59dbe3ed0dc0a2290ea5c9/lib/config/presets/internal/monorepo.ts#L6